### ISSUE TITLE
Sensors not showing

### ISSUE THREAD

**Issue by kjetilpp** [custom-components/ble_monitor#487]
Hello! 
Are running HA core-2021.8.8 on Raspberry Pi 4 Model B, 8GB RAM with ble_monitor on 5.0.0

Just received 2 Teltonika sensors. Blue Puck T and Blue Coin T. They are active and i can see the temperature using a BLE Scanner on my phone. But the ble_monitor cant find them. Discovery and active scan is enabled. 
The sensors are about 2 meters from the Pi. Need some suggestions on what I can check next.
![Capture](https://user-images.githubusercontent.com/5620578/131339857-674e145e-ab2e-4900-b2bc-6f6d78ad8de7.PNG)

**Comment by Ernst79** [comment]
Can you enable debug logging and check if you get messages with a long string of numbers and letters in the HA log. Leave report_unknown to Teltonika, as you have now. 

```
logger:
  default: warn
  logs:
    custom_components.ble_monitor: info
```

Please post a few of these messages if you get these. 
If not, can you make a hcidump? Note that this is not easily possible if you are running Home Assistant Operating System. Let me know if that is the case, we have other ways to get the info.

**Comment by kjetilpp** [comment]
`2021-08-30 14:47:10 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -47, MAC: E2:09:80:86:76:BB, ADV: 05166e2ad0090b0943205420383031424239d1
2021-08-30 14:47:11 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -79, MAC: EA:93:83:85:71:3B, ADV: 05166e2ac2020b0950205420383034343732b1
2021-08-30 14:47:12 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2ad0090b0943205420383031424239ce
2021-08-30 14:47:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2ad0090b0943205420383031424239ce
2021-08-30 14:47:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -76, MAC: EA:93:83:85:71:3B, ADV: 05166e2ad5020b0950205420383034343732b4
2021-08-30 14:47:16 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2ad0090b0943205420383031424239ce
2021-08-30 14:47:17 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -79, MAC: EA:93:83:85:71:3B, ADV: 05166e2aee020b0950205420383034343732b1
2021-08-30 14:47:18 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2ad0090b0943205420383031424239ce
`

This log?

**Comment by Ernst79** [comment]
yes, I will have a look

**Comment by Ernst79** [comment]
Ok, your logs show one MAC address. The name of this device is "C T 801BB9" (with MAC E2:09:80:86:76:BB), are you able to find out which one this is? If it is the coin, I can add it as a new device. I suspect it is the coin, given the first letter C. 

Your logs only show one MAC. Can you also provide a log of the other sensor or, if not found, could you show me a printscreen of your phone while scanning the other sensor? The Blue Puck T is already implemented, but it is checking the name. Perhaps it has changed.

**Comment by kjetilpp** [comment]
Yes C T 801BB9 is the coin.
EA:93:83:85:71:3B is the other mac, look here

`2021-08-30 15:07:46 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -84, MAC: EA:93:83:85:71:3B, ADV: 05166e2acdfb0b0950205420383034343732ac
2021-08-30 15:07:46 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -48, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d0
2021-08-30 15:07:47 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -47, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d1
2021-08-30 15:07:48 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -87, MAC: EA:93:83:85:71:3B, ADV: 05166e2ae6fb0b0950205420383034343732a9
2021-08-30 15:07:49 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -51, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239cd
2021-08-30 15:07:50 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:07:51 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -83, MAC: EA:93:83:85:71:3B, ADV: 05166e2a0cfc0b0950205420383034343732ad
2021-08-30 15:07:51 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -44, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d4
2021-08-30 15:07:52 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -85, MAC: EA:93:83:85:71:3B, ADV: 05166e2a12fc0b0950205420383034343732ab
2021-08-30 15:07:52 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:07:53 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -44, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d4
2021-08-30 15:07:54 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -84, MAC: EA:93:83:85:71:3B, ADV: 05166e2a25fc0b0950205420383034343732ac
2021-08-30 15:07:54 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:07:55 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -78, MAC: EA:93:83:85:71:3B, ADV: 05166e2a31fc0b0950205420383034343732b2
2021-08-30 15:07:55 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -45, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d3
2021-08-30 15:07:57 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -44, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d4
2021-08-30 15:07:58 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:07:59 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -44, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d4
2021-08-30 15:08:00 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:08:02 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -49, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239cf
2021-08-30 15:08:04 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -83, MAC: EA:93:83:85:71:3B, ADV: 05166e2a83fc0b0950205420383034343732ad
2021-08-30 15:08:04 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:08:05 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -45, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d3
2021-08-30 15:08:06 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:08:07 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -44, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d4
2021-08-30 15:08:08 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -82, MAC: EA:93:83:85:71:3B, ADV: 05166e2aa8fc0b0950205420383034343732ae
2021-08-30 15:08:08 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:08:09 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -80, MAC: EA:93:83:85:71:3B, ADV: 05166e2aaefc0b0950205420383034343732b0
2021-08-30 15:08:09 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -44, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239d4
2021-08-30 15:08:10 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -50, MAC: E2:09:80:86:76:BB, ADV: 05166e2aca090b0943205420383031424239ce
2021-08-30 15:08:11 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -76, MAC: EA:93:83:85:71:3B, ADV: 05166e2ac1fc0b0950205420383034343732b4`

**Comment by Ernst79** [comment]
Thanks, they apparantly have changed the name. This one is now called "P T 804472" in stead of PUCK_T1. I'll make a fix and add the coin.

**Comment by kjetilpp** [comment]
Yes, the number is a variable for each device. The number is printed on the coin/puck.

**Comment by Ernst79** [comment]
5.1.1-beta has been released, which should add support for the coin and fix the puck being recognized. Please enable beta versions in HACS to test.

**Comment by kjetilpp** [comment]
It works 😄 
![Capture](https://user-images.githubusercontent.com/5620578/131361576-e46a62ca-8479-47d0-b4ec-e634665789cf.PNG)

**Comment by Ernst79** [comment]
is it freezing?

**Comment by kjetilpp** [comment]
Yes. But RSSI -89, so is probably just poor range?

**Comment by Ernst79** [comment]
Yes, that is poor signal strength. Can be due to the temperature, affects battery voltage.

or did you put it in the freezer? A freezer will also block the signal due to the steel casing.

but there can be more reasons for bad signal strength, like an SSD drive close to the Bluetooth radio.

**Comment by kjetilpp** [comment]
Yes, i do have it in the freezer, the battery is not an issue, they are made for it 😆 
My Pi also have an SSD and Zigbee 🆘 

So are looking for a solution 🤓

**Comment by Ernst79** [comment]
An option is to use an usb extension cable and external dongle, to create distance between the SSD and Bluetooth radio. Or an extension cable on the SSD. But I would first test if the issue is with the freezer steel casing, or on the Bluetooth radio side, by placing the sensor outside the freezer, on the same spot, and check if it improves or not.

**Comment by Ernst79** [comment]
I'm closing this issue. Fixed in 5.1.1-beta.
